src/components/datePicker/tests/DatePickerReducers.spec.ts

```diff
@@ -181,24 +183,33 @@ describe('Date picker', () => {
       expect(otherDatePicker.appliedUpperLimit).not.toBe(otherDatePicker.upperLimit);
     });
 
-    it('should return the state with the new lower limit for the date picker with the action id when the action is ' +
-      '"CHANGE_LOWER_LIMIT"', () => {
-        const oldState: IDatePickerState[] = [
-          _.extend({}, BASE_DATE_PICKER_STATE, { id: 'some-date-picker2' }),
-          _.extend({}, BASE_DATE_PICKER_STATE),
-          _.extend({}, BASE_DATE_PICKER_STATE, { id: 'some-date-picker3' }),
-        ];
-        const action: IReduxAction<IChangeDatePickerPayload> = {
-          type: DatePickerActions.changeLowerLimit,
-          payload: {
-            id: 'some-date-picker',
-            date: new Date(new Date().setHours(4, 4, 4, 4)),
-          },
-        };
-        const datePickerState: IDatePickerState = _.findWhere(datePickersReducer(oldState, action), { id: action.payload.id });
-        expect(datePickerState.lowerLimit).toBe(action.payload.date);
-        expect(datePickerState.selected).toBe('');
-      });
+    it('should clear all clearable date pickers starting with the action id if the action is "CLEAR_SELECTION"', () => {
+      const oldState: IDatePickerState[] = [
+        _.extend({}, BASE_DATE_PICKER_STATE, { id: 'some-date-picker2', isClearable: true }),
+        _.extend({}, BASE_DATE_PICKER_STATE),
+        _.extend({}, BASE_DATE_PICKER_STATE, { id: 'other-id' }),
+      ];
+      const action: IReduxAction<IDatePickerPayload> = {
+        type: DatePickerActions.clear,
+        payload: {
+          id: 'some-date-picker',
+        },
+      };
+      const datePickersState: IDatePickerState[] = datePickersReducer(oldState, action);
```

Comment:
if you name datePickersState to `newState` i think your test will be more readable overall